AmazonChimeSDK/AmazonChimeSDK/AmazonChimeSDK.h

```diff
@@ -0,0 +1,15 @@
+//
+//  AmazonChimeSDK.h
+//  AmazonChimeSDK
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+#ifndef AMAZON_CHIME_SDK_H
```

Comment:
Not familiar about it. Is this `if not defined` condition check mandatory?

Reply:
It is not mandatory but it's good practice to do so. The reason for doing so is to make sure the header file gets instantiated only once. So if multiple "child" headers or implementation files import the header, it won't need to be run again so it's already defined.